src/statsmodels/formula.jl

```diff
@@ -419,7 +419,7 @@ function droprandomeffects(trms::Terms)
     else
         # the rows of `trms.factors` correspond to `eterms`, the columns to `terms`
         # After dropping random-effects terms we drop any eterms whose rows are all false
-        ckeep = !retrms                 # columns to retain
+        ckeep = map(!, retrms)                 # columns to retain
```

Comment:
Nice, I like this better than the alternative `(!).(retrms)`, which also works but looks bizarre.